.github/workflows/editor-swagger-io.yml

```diff
@@ -1,20 +0,0 @@
-name: Validate OpenAPI schema - editor.swagger.io
```

Comment:
Not sure to understand why this workflow is removed. Could you give us a bit of context?

Reply:
Thank you for those details :pray: 

Could you rebase main in your PR? Apparently their is a conflict in the swagger file